calculator_gui/tasks/lagrange_coeff.py

```diff
@@ -0,0 +1,33 @@
+import math
+
+"""
+NAME - SHASHVAT JAIN 
+ADMISSION NO. - 20JE0897
+"""
+
+
+def sc(z):
+    if z > 0:
+        return (1 - math.cos(math.sqrt(z))) / z
+    if z < 0:
+        return (math.cosh(math.sqrt(-z)) - 1) / (-z)
+    else:
+        return 1 / 2
+
+
+def ss(z):
+    if z > 0:
+        return (math.sqrt(z) - math.sin(math.sqrt(z))) / (math.sqrt(z)) ** 3
+    if z < 0:
+        return (math.sinh(math.sqrt(-z)) - math.sqrt(-z)) / (math.sqrt(-z)) ** 3
+    else:
+        return 1 / 6
+
+
+def main(m, x, t, r, ro, a):
```

Comment:
What are the data types of `m, x, t, r, ro, a`. Add [type hints](https://docs.python.org/3/library/typing.html)